assets/src/edit-story/components/inspector/inspectorProvider.js

```diff
@@ -38,15 +39,19 @@ function InspectorProvider({ children }) {
   const [users, setUsers] = useState([]);
   const [statuses, setStatuses] = useState([]);
   const [inspectorContentHeight, setInspectorContentHeight] = useState(null);
+  const inspectorContentRef = useRef();
 
   const [isUsersLoading, setIsUsersLoading] = useState(false);
   const [isStatusesLoading, setIsStatusesLoading] = useState(false);
 
-  // TODO Implement `useResizeObserver` once either branch is merged
   const setInspectorContentNode = useCallback((node) => {
-    setInspectorContentHeight(node.clientHeight);
+    inspectorContentRef.current = node;
```

Comment:
Is it 100% certain that the `setInsepctorContentNode` will be called in the render-phase before we get to the side-effect of the `useResizeEffect`?

Reply:
`setInspectorContentNode` is used here:
```jsx
<InspectorBackground ref={setInspectorContentNode}>
```
and `useResizeEffect` uses `useEffect` which happens after render, so yes, in this case ref will be set before side-effects.